internal/scorecard/alpha/testpod.go

```diff
@@ -47,12 +47,38 @@ func getPodDefinition(test Test, o Scorecard) *v1.Pod {
 					Image:           "quay.io/operator-framework/scorecard-test:dev",
 					ImagePullPolicy: v1.PullIfNotPresent,
```

Comment:
not a blocker too. However, this one shows that could be covered by unit tests as well. See: https://coveralls.io/builds/30552210/source?filename=internal/scorecard/alpha/testpod.go